examples/extensions/r2/deflat_r2.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python3
+#
+# Cross Platform and Multi Architecture Advanced Binary Emulation Framework
+#
+
+import sys
+
+sys.path.append('..')
+
+from qiling import Qiling
+from qiling.const import QL_VERBOSE
+from qiling.extensions.r2 import R2Qiling
+
+
+
+if __name__ == "__main__":
+    # a program obfuscated by OLLVM CFF flatten, which should print 4 when argv[1] is 1
+    ql = R2Qiling(['rootfs/x86_linux/bin/test_fla_argv', '1'], 'rootfs/x86_linux', verbose=QL_VERBOSE.DEFAULT)
```

Comment:
Source code of this file and steps to compile.

Reply:
```c
#include <stdio.h>
#include <stdlib.h>
                 
unsigned int target_function(unsigned int n)
{                
  unsigned int mod = n % 4;
  unsigned int result = 0;                                                                                                         
                 
  if (mod == 0) result = (n | 0xBAAAD0BF) * (2 ^ n);
                 
  else if (mod == 1) result = (n & 0xBAAAD0BF) * (3 + n);
                                                                                                                                   
  else if (mod == 2) result = (n ^ 0xBAAAD0BF) * (4 | n);                                                                          
                                                                                                                                   
  else result = (n + 0xBAAAD0BF) * (5 & n);                                                                                        
                                                                                                                                   
  return result;                                                                                                                   
}                                                                                                                                  
                                                                                                                                   
int main(int argc, char **argv) {                                                                                                  
   int n;                                                                                                                          
   if (argc < 2) {                                                                                                                 
      n = 0;                                                                                                                       
   } else {                                                                                                                        
      n = atoi(argv[1]); nptr:                                                                                                     
   }                                                                                                                               
   int val = target_function(n);                                                                                                   
   printf("%d\n", val); format:                                                                                                    
   return 0;                                                                                                                       
}
```

`clang -m32 -mllvm -fla test.c -o test_fla_argv`

Reply:
@wtdcode Should I upload the binary to rootfs or put source code in the comment of this example?

Reply:
> @wtdcode Should I upload the binary to rootfs or put source code in the comment of this example?

`examples/src/linux`